### DISCUSSION TITLE
Aris Studio Development - Custom font feature

### DISCUSSION THREAD

**Discussion by kiraio-moe** [Tualin14/baPlayer#15] [Development]
Hi, @Tualin14. As you may know, I disable the custom font feature [`9738d654`](https://github.com/Tualin14/ArisStudio/commit/9738d6546540125cc585dcdc0ee487ce209394ef#diff-76b752d3e195de99f9f56d47b5dc430d0dc1b514088bc00b7758d1ee19214ddbR74) because I think it's pretty useless to use user installed fonts. Knowing the bug #11 that not all fonts is returned, it's also impacting performance because when the game is started it's populating installed fonts then add them to dropdown list causing the scene loading more long. Because of the dropdown list/content is huge (depending on user installed fonts), there will be a slightly delay when expanding the dropdown.

To fix this issue, I have an idea to just include Latin and [CJK](https://en.wikipedia.org/wiki/CJK_characters) fonts that similar to Blue Archive is used + licensed under SIL Open Font License (or any other Open Source license).

What do you think?

**Comment by ttttttttttttttllllllllllllll** [comment]
I have a plan for this is to use different fonts for different language settings, so far I have a choice for Chinese [Resource Han Rounded SC regular](https://github.com/CyanoHao/Resource-Han-Rounded), I was about to ask if you have any English recommendations?

**Reply by kiraio-moe** [reply]
> I was about to ask if you have any English recommendations?

Still finding it ATM. There's tons of fonts on my machine to be checked.
